changelog/Microsoft.DirectoryServices.json

```
@@ -26,6 +26,40 @@
       "WorkloadArea": "Identity and access",
       "SubArea": "Directory management"
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "eaf1d043-b258-4b47-afce-9183e50aacb2",
+          "ApiChange": "Resource",
+          "ChangedApiName": "federatedIdentityCredentials",
+          "ChangeType": "Addition",
+          "Description": "Added the **federatedIdentityCredentials** relationship to the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-1.0) resource.",
+          "Target": "application"
+        },
+        {
+          "Id": "eaf1d043-b258-4b47-afce-9183e50aacb2",
+          "ApiChange": "Resource",
+          "ChangedApiName": "federatedIdentityCredential",
+          "ChangeType": "Addition",
+          "Description": "Added the [federatedIdentityCredential](https://docs.microsoft.com/en-us/graph/api/resources/federatedIdentityCredential?view=graph-rest-1.0) resource",
```

Suggested enhancement
```suggestion
          "Description": "Added the [federatedIdentityCredential](https://docs.microsoft.com/en-us/graph/api/resources/federatedIdentityCredential?view=graph-rest-1.0) resource and associated methods. A federated identity credential is a secure method for managing an application's credentials that allows a tenant's cloud applications to access Azure AD without using secrets and certificates.",
```